test-data/unit/deps-types.test

```diff
@@ -566,6 +665,20 @@ class I: pass
 <mod.I> -> <m.P>, m, m.P
 <mypy_extensions.TypedDict> -> m
 
+[case testAliasDepsTypedDictFunctional]
+from mypy_extensions import TypedDict
+from mod import I
+A = I
+P = TypedDict('P', {'x': A})
+[file mod.py]
+class I: pass
+[builtins fixtures/dict.pyi]
+[out]
+<m.A> -> m
```

Comment:
Like above, does this actually test that the dependency is generated, or would the dependency generate for just `A = I`, without the typed dict definition?